src/server/ns_turn_server.c

```diff
@@ -5010,7 +5017,7 @@ void init_turn_server(turn_turnserver *server, turnserver_id id, int verbose, io
 
   server->response_origin_only_with_rfc5780 = response_origin_only_with_rfc5780;
 
-  server->respond_http_unsupported = respond_http_unsupported;
```

Comment:
Why change this?

Reply:
Nice find - must have been a bad resolution of a merge conflict.  I'll fix it up.